study.md

```diff
@@ -48,7 +53,10 @@ in Rails.
 In your own words, define what the router does in Rails.
 
 ```md
-<!-- your answer here -->
+The router is the doorman it looks at http requests and determine which
+controller action is equivalent to the request.  In rails there are seven
+types of verbs while http request has 4.  Rails verb is a bit more intuitive
```

Comment:
Not exactly correct, but pretty close. There are only four verbs. VERB + URL make a request. There are seven standard combinations of VERB + URL that make up the default controller actions.